_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,276 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] with a [visible][] [text node][] if:
+
+- **changed:** The `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+**Note:** The test targets for this rule are elements that have their text content change multiple times, on a page with further content, if that change happens without user intervention.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. It is however a much lesser accessibility issue, and without any time limit this rule would be virtually untestable.
+
+## Expectation
+
+For each test target there exists an [instrument][] in the same [web page](#web-page-html), or in another [web page](#web-page-html) that is [linked][hyperlink] from the [web page](#web-page-html) of the test target, to achieve one of the following objectives:
```

Comment:
Another issue, what if the instrument isn't in the page, but you can do something to get it there, such as opening a modal?

Reply:
A modal is still in the same web page. I've added a note to indicate the user might have to perform an action for the instrument to become available.

Reply:
I disagree with that, which if nothing else suggests "in the same web page" is ambiguous. This needs to be fixed.

Reply:
Just to be clear: which part do you disagree with? That the modal is on the same web page? Or with the contents of the note?
(if it's the former, can you tell me why you disagree with it?)

Reply:
I disagree that the content on a modal is in the same web page. It isn't until someone opens the modal.